File: 백준/Bronze/9093. 단어 뒤집기/단어 뒤집기.py
import sys
line = int(sys.stdin.readline())


for i in range(line):
    alist=sys.stdin.readline().split()
    word=""
    for u in range(len(alist)):
        now=alist[u]
        for o in reversed(now):
             word+=o
        word+=" "
    print(word)

File: 백준/Bronze/9093. 단어 뒤집기/단어 뒤집기.java

    import java.io.IOException;
    import java.io.BufferedReader;
    import java.io.InputStreamReader;
    import java.util.ArrayList;

    public class Main {
        public static void main(String[] args) throws IOException {

            BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
            int len = Integer.parseInt(reader.readLine());
            ArrayList<String> command= new ArrayList<>();
            for (int i = 0; i < len; i++) {
                String line = reader.readLine();
                String[] parts = line.split(" ");
                StringBuffer result = new StringBuffer();
                for(String part: parts){
                    StringBuffer rr =new StringBuffer(part);
                    result.append(rr.reverse().toString()).append(" ");
                }

                System.out.println(result);
                }



            }
            }



